.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = tab
+indent_size = 4
```

Comment:
Wouldn't the indent size be a personalized choice? e.g. I'm sane and use 8 space tabs like a normal person

Reply:
First of all, what kind of psycho uses an 8 space tab?! :P 
But yes, that line should actually be omitted as it relates to using spaces, not tabs, for indentation.